subprojects/docs/src/docs/userguide/what_is_gradle.adoc

```diff
@@ -12,133 +12,114 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+[[gradle_overview]]
 [[what_is_gradle]]
 = What is Gradle?
 
-[[gradle_overview]]
-== Overview
-
-Gradle is an open-source https://en.wikipedia.org/wiki/Build_automation[build automation] tool that is designed to be flexible enough to build almost any type of software. The following is a high-level overview of some of its most important features:
+Gradle is an open-source https://en.wikipedia.org/wiki/Build_automation[build automation] tool
+flexible enough to build almost any type of software.
```

Comment:
```suggestion
Gradle is an open-source https://en.wikipedia.org/wiki/Build_automation[build automation] tool
flexible enough to build almost any type of software.
This flexibility is the result of Gradle making few assumptions about what you're trying to build or how to build it.
```
@tresat What do you think of this alternate location -- right in the introduction? https://github.com/gradle/gradle/pull/22277#discussion_r986795554